### DISCUSSION TITLE
Trying to remove jutter from video frames.

### DISCUSSION THREAD

**Discussion by ghohenzollern** [ImageMagick/ImageMagick#6136] [Help] [answered]
### ImageMagick version

7.1.0

### Operating system, version and so on

Windows 10

### Description

Hello!

I have a 60i video where the even fields are all one scan line too low, leading to some serious jaggies.  I can't find a filter in VirtualDub that will fix this, so I thought before going to the trouble to write my own, I'd look to see what might already be out there.  I found and am trying ImageMagick at the moment.  From VirtualDub, I've separated the fields into separate frames and saved them as a series of .pngs named 000000.png 000001.png, etcetera.  From reading the ImageMagick documentation, I got the impression something like this would work:

magick mogrify *0.png -crop 720x239+0+1 -path output
magick mogrify *1.png -crop 720x239+0+0 -path output
magick mogrify *2.png -crop 720x239+0+1 -path output
magick mogrify *3.png -crop 720x239+0+0 -path output
magick mogrify *4.png -crop 720x239+0+1 -path output
magick mogrify *5.png -crop 720x239+0+0 -path output
magick mogrify *6.png -crop 720x239+0+1 -path output
magick mogrify *7.png -crop 720x239+0+0 -path output
magick mogrify *8.png -crop 720x239+0+1 -path output
magick mogrify *9.png -crop 720x239+0+0 -path output

What I am trying to do is to crop one line off the top of all the even frames and off the bottom of all the odd frames.  But no files are placed in the output folder.  I've also tried not specifying a path which supposedly means the files are to be modified in place.  And the files do get accessed, their dates change, but their contents are unmodified.  They're all still 720x240, rather than 720x239.  Any idea what I'm doing wrong?

**Comment by fmw42** [comment, marked as answer]
Try this syntax

`magick mogrify -path output -crop 720x239+0+1 +repage *0.png `

**Reply by ghohenzollern** [reply]
Well, I typed a response, but then when I clicked "mark as answer" my response disappeared, and I"m not going to type it again.  Long and short of it I was trying to spend less time learning imagemagick than it would have taken me to write the filter I needed, and I rushed and missed an important part of the documentation.  But this is the answer.  Yay!
